src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -286,6 +338,713 @@ void AssimpImporterTest::openDataFailed() {
     CORRADE_COMPARE(out.str(), "Trade::AssimpImporter::openData(): loading failed: No suitable reader found for the file format of file \"$$$___magic___$$$.\".\n");
 }
 
+namespace {
+
+/* This does not indicate general assimp animation support, only used to skip
+   tests on certain versions and test files. */
+bool supportsAnimation(const std::string& fileName) {
+    /* 5.0.0 supports all of Collada, FBX, glTF */
+    if(ASSIMP_IS_VERSION_5)
+        return true;
+
+    /* splitExtension returns empty extensions for files starting with a dot */
+    const std::string extension = Utility::Directory::splitExtension("x" + fileName).second;
+    const bool isGltf = extension == ".gltf";
+    if(isGltf) return false;
```

Comment:
```suggestion
    if(Utility::String::hasSuffix(fileName, ".gltf") return false;
```

Or switch the whole thing to a `StringView` and use [hasSuffix()](https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1BasicStringView.html#a403edf0edbe56aa02add16ee25df91f1).

(Commenting just because I was doing a very similar thing elsewhere just now :))

Reply:
Perfect, that just removed 2 lines of code and a variable 👌